lib/rules/button-icon-a11ytitle.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
     },
     fixable: null,
     messages: {
-      'button-icon-a11ytitle': `Icon-only button requires 'a11yTitle' or 'aria-label' describing the action of the button.`,
+      'button-icon-a11ytitle': `Icon-only button requires 'a11yTitle', 'aria-label', or 'tip' describing the action of the button. Use 'tip' if it would be valuable for the description to appear on hover.`,
+      'button-icon-custom-tip': `Because 'tip' is an object, an 'a11yTitle' or 'aria-label' describing the action of the button is required to help users of assisstive technologies.`,
```

Comment:
```suggestion
      'button-icon-custom-tip': `Because 'tip' is implemented as a custom object an `aria-label` is not automatically added to button. Include an accompanying 'a11yTitle' or 'aria-label' describing the action of the button to help users of assistive technologies.`,
```